src/core/Microsoft.PowerToys.Settings.UnitTest/ViewModelTests/PowerLauncherViewModelTest.cs

```diff
@@ -100,14 +100,6 @@ public void Hotkeys_ShouldUpdateHotkeys()
                 false,
                 (int)Windows.System.VirtualKey.A
                 );
-            AssertHotkeySettings(
-                mockSettings.Properties.open_console,
-                false,
-                false,
-                true,
-                false,
-                (int)Windows.System.VirtualKey.D
-                );
```

Comment:
According to @laviusmotileng-ms .  We didn't intent to remove the OpenConsole method.  It was accidentally removed as part of resolving a merge conflict.   I think we should add it back in.  Sorry for the miscomunication.  I did a PR #5191, but it also needs to be updated.

Reply:
I'll close this PR then.